src/components/organisms/Modal.js

```diff
@@ -35,19 +39,22 @@ const Modal = ({
           {children}
         </main>
         <footer>
-          {canCancel && (
+          {cancelHandler && (
             <Button
               type="button"
               text={cancelText}
               variant="secondary"
               clickHandler={handleCancel}
             />
           )}
-          <Button
-            type="submit"
-            text={confirmText}
-            variant="primary"
-          />
+          {buttons}
```

Comment:
the `buttons` elements if provided will display between cancel and confirm buttons. I may just remove buttons altogether and have the `buttons` prop populate this entire footer portion of the modal component. Not sure yet on that one.